### DISCUSSION TITLE
Copilot student account application rejected

### DISCUSSION THREAD

**Discussion by israelbolvr** [community/community#113788] [GitHub Education]
### Hi everyone,

I applied with my university student email, image of my University ID and every other information in the form on 28/11/2023, but, my submission was rejected.

![Captura de tela 2024-03-23 091549](https://github.com/community/community/assets/112780022/26ec75a9-0d1a-49a7-8811-60c7659fc96d)

**Comment by samus-aran** [comment]
Hi @israelbolvr 👋

You do appear to have your academic benefits. If you are having trouble enabling Copilot I would suggest following these instructions:

To enable Copilot, go to your account settings and under Code, planning, and automation section, select Copilot. You'll see "You're eligible to use copilot for free" and a button to claim the offer.

If you still unable to enable Copilot please raise a ticket with [Education support ](https://support.github.com/contact/education)using the dropdown "I am a verified student having trouble redeeming my Global Campus offers". 👍

Hope this helps! Let us know how you get along 👍
